Remotion/ObjectBinding/Web/Res/Themes/NovaViso/HTML/BocCheckBox.css

```diff
@@ -6,56 +6,24 @@
   align-items: center;
 }
 
-.bocCheckBox input
-{
-  position: absolute;
-  z-index: var(--z-index-hidden);
-}
-
 .bocCheckBox span.description
 {
   display: inline-flex;
   align-items: center;
 }
 
 .bocCheckBox img,
-.bocCheckBox input + span.checkbox-visualizer::before
+.bocCheckBox input + span.checkbox-visualizer:before
```

Comment:
we no longer have this css class. Also, the alignment stuff should (probably?) come from Common.css. It really depends on which parts can be seen as generic and which parts are specific. a good hint is if the same rules are needed for BocBoolean, BocCheckbox and in BOclist. Also we should be consistent between radiobutton and checkbox